go.mod

```diff
@@ -18,18 +18,18 @@ require (
 	github.com/prometheus/prometheus v1.8.2-0.20210518124745-6eeded0fdf76 // v1.8.2 is misleading as Prometheus does not have v2 module. This is pointing to v2.26.1, the same as in prometheus-operator v0.47.1
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.21.0
-	k8s.io/apiextensions-apiserver v0.21.0
-	k8s.io/apimachinery v0.21.0
-	k8s.io/apiserver v0.21.0
+	k8s.io/api v0.21.2
+	k8s.io/apiextensions-apiserver v0.21.2
+	k8s.io/apimachinery v0.21.2
+	k8s.io/apiserver v0.21.2
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog/v2 v2.8.0
 	k8s.io/kube-aggregator v0.20.0
 	k8s.io/metrics v0.19.4
 )
 
 replace (
-	k8s.io/api => k8s.io/api v0.21.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.21.0
-	k8s.io/client-go => k8s.io/client-go v0.21.0
+	k8s.io/api => k8s.io/api v0.21.2
```

Comment:
I think we need the replace only for the client-go library.

Reply:
correct! updated.